Lib/json/__init__.py

```diff
@@ -342,6 +343,14 @@ def loads(s, *, encoding=None, cls=None, object_hook=None, parse_float=None,
                             f'not {s.__class__.__name__}')
         s = s.decode(detect_encoding(s), 'surrogatepass')
 
+    if encoding is not _sentinel:
+        import warnings
+        warnings.warn(
+            "passing 'encoding' as a keyword argument is deprecated since"
+            " Python 3.1 and will be ignored.", DeprecationWarning,
```

Comment:
```suggestion
            " Python 3.1 and will be removed in Python 3.10.", DeprecationWarning,
```

Reply:
I've done both, as "is ignored" is a current fact. "Deprecated" is not enough to know what the current behavior is. It could be deprecated but still having effect.